tests/PHPC-bug00631.phpt

```diff
@@ -0,0 +1,42 @@
+--TEST--
+PHPC-631: UTCDateTime::toDateTime() may return object that cannot be serialized
+--SKIPIF--
+<?php if (!defined("HHVM_VERSION_ID")) exit("skip HHVM uses 'UTC' instead of '+00:00'"); ?>
```

Comment:
I realized after the fact that this was copied from https://github.com/mongodb/mongo-php-driver/pull/416. Changing the message makes sense.